ConferenceService/Configuration/ConferenceDatabaseSettings.cs

```diff
@@ -1,13 +1,32 @@
 ﻿namespace ConferenceService.Configuration
 {
+    using System;
+
     /// <summary>
     /// The settings required to setup a successful connection
     /// to the mongo db instance.
     /// </summary>
     public class ConferenceDatabaseSettings
     {
-        public string ConferenceCollectionName { get; set; }
-        public string ConnectionString { get; set; }
-        public string DatabaseName { get; set; }
+        private const string MongoDbHostDefault = "mongodb://localhost:27017";
+        private const string MongoDbHostEnvironmentVar = "MONGO_DB_HOST";
+
+        public string ConnectionString
+        { 
+            get 
+            {
+                var mongoDbHost = Environment.GetEnvironmentVariable(MongoDbHostEnvironmentVar);
```

Comment:
Not sure if this call is expensive or cheap, but I don't think it's a good idea to make it every time we need a connection string. I think we can cache the value in a field after the first get.

Reply:
@ghewitt This is a singleton, so it would only be instantiated on the initial call.